include/nfc/tnep/tag.rst

```diff
@@ -0,0 +1,142 @@
+.. _tnep_tag_readme:
+
+TNEP for tag device
+###################
+
+This library provides an implementation of the Tag NDEF Exchange Protocol (TNEP) for an NFC tag device.
+
+It is used in the :ref:`nfc_tnep_tag` sample.
+
+Initialization
+**************
+
+Before using the TNEP library in an NFC tag application, you must configure the NDEF message buffers and define the TNEP services.
+
+Configuring NDEF message buffers
+================================
+
+The tag library communicates with the tag using NDEF message buffers.
```

Comment:
With "tag emulation library", do you mean the Type 2 Tag or Type 4 Tag library in nrfxlib?
We don't use that term there anymore, but we could add a link here to clarify.

Reply:
Hmm it is referring to the Type 4 Tag library, but just wanted to avoid this: "The tag library communicates with the tag" which I find confusing. This "Tag library" is a simplified term for the "TNEP Tag library". That is what needs to communicate with the Type 4 Tag lib from nrfxlib. So, yes let's add the link to the Type 4 Tag lib clarify